Server/DB/index.js

```diff
@@ -0,0 +1,15 @@
+require("dotenv").config();
+const mongoose = require("mongoose");
+
```

Comment:
also, we might have to start  looking at using  async-await to handle promises 

just a personal preference I feel it makes the code way cleaner.

as always nice work you dey burst brain

Reply:
I agree with you, I think the only place I didn’t use a async and await was in the db file. I’ll refactor it ASAP. Thanks again 🙏